controllers/example_controller/keyboard_controller.py

```diff
@@ -22,6 +22,18 @@
     "angle_unit": (ord("B"), ord("B")),
 }
 
+# Map robot zone to control set. Defaults to straight mapping, but provides a
+# place to remap other zones if desired. Modify the keys of the dictionary to
+# the zones you want to use.
+ZONE_MAP = {
```

Comment:
Shouldn't this be:
```
0: 0,
1: 1,
```

So zone 0 has WSAD controls and zone 1 has IJKL controls?

Otherwise it'll change the behavior of the robot

Reply:
Gah, oops. Yeah, it was like that and then I was testing it and apparently committed the test code.

9988927